src/js/Templates/WebView/WebView.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import AppHeader from '../../containers/Header';
+
+class WebView extends React.Component {
+    render() {
+        return (
+            <div className="webView">
+                <AppHeader backLink="/" menuName="Apps" icon="false"/>
```

Comment:
I assune WebengineAppContainer was not included in this component because the iframe needs to be running when not in view?

Reply:
Exactly, the iframe has to be outside the route context so it can run when we are in a menu etc.